packages/frontend/src/lib/BootcEmptyScreen.svelte

```diff
@@ -5,5 +5,5 @@ import BootcIcon from './BootcIcon.svelte';
 
 <EmptyScreen
   icon="{BootcIcon}"
-  title="No bootable container builds found"
```

Comment:
I think the lower case was actually correct here. Bootable container is not a trademark or title for something, it's just a name like 'pod',  'image' or 'container' where we don't capitalize.

Unfortunate that we need to show builds here, otherwise we could call these disk images. Or maybe we should shift and start making this page more like the others and just show disk images??